src/exoscale/tools/project/api/tasks.clj

```diff
@@ -129,29 +129,27 @@
       :ref (binding [td/*the-dir* dir]
              (exoscale.tools.project.api.tasks/task (assoc opts :id ref) opts)))))
 
+(defn- filter-dir?
+  [task sub-edn]
+  (let [when' (get sub-edn (:when task) true)
```

Comment:
The idea behind `:when` was to encode opt-in task participation, by defaulting to `true` it would become opt-out and thus make `:unless` irrelevant.

Leaving the two exclusive options exclusive feels like the easiest thing to grasp.

Reply:
yeah it's a mistake in the pr, I meant to have :when be any? if no value is present, not to have when resolved value to default to true. Basically the underlying behavior should not change.

Reply:
should be fixed now